content/21.nutraceuticals.md

```diff
@@ -1,50 +1,97 @@
-## Nutraceuticals
+## Dietary Supplements and Nutraceuticals Under Investigation for COVID-19 Prevention and Treatment
 
 ### Abstract
 
+The coronavirus disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has caused global disruption and a significant loss of life.
+Scientists and physicians around the world have worked to discover and develop novel treatments or repurpose existing treatments in order to identify potential prophylactic and therapeutic agents to stem the spread of SARS-CoV-2 and reduce the devastation of this pandemic.
+Dietary supplements and nutraceuticals are also being investigated in this context.
+Despite a lack of evidence to support their efficacy or safety for the treatment or prevention of COVID-19, the beginning of the pandemic led to increased sales of vitamins, supplements, nutraceuticals, and various other products purporting 'immune-boosting' effects.
+Considerable interest amongst the scientific community and emerging evidence has led to further investigation of these products. 
+In this review, we critically appraise the evidence surrounding dietary supplements and nutraceuticals for the prophylaxis and treatment of COVID-19.
+These include vitamin C, vitamin D, omega 3 polyunsaturated fatty acids (PUFA), probiotics, and zinc.
+These were selected as they are currently under clinical investigation.
+There is increasing evidence to suggest that vitamin D supplementation may be beneficial due to plausible associations between vitamin D deficiency with COVID-19 incidence and possibly disease severity.
+A growing number of scientific organizations are considering or advising for increased vitamin D supplementation to those at high risk of COVID-19.
+Overall, caution is warranted as further evidence and clinical trials are required before conclusive evidence-based recommendations can be formulated for any specific dietary supplements or nutraceuticals.
+Consumers should also be aware of misinformation and false promises surrounding some supplement products, which may lack safety and efficacy evidence due to poor regulation by authorities.
+However, considering that nutritional status plays a significant role in patient outcomes, it would be pertinent to advise the general public to follow a healthy diet and lifestyle to prevent nutrient deficiencies and to maintain a healthy immune system.
```

Comment:
```suggestion
However, considering that nutritional status plays a significant role in patient outcomes, it would be prudent to advise the general public to follow a healthy diet and lifestyle to prevent nutrient deficiencies and to maintain a healthy immune system.
```